docs/hugo.toml

```diff
@@ -14,7 +14,6 @@ enableMissingTranslationPlaceholders = false
 [params]
 
     # Source Code repository section
-    description = "TRISA Documentation"
```

Comment:
This was removed to resolve a build error. An update to the theme required moving the description to `_index.en.md`. This was previously done, so deleting this line was the only requirement.